index.html

```diff
@@ -17,24 +17,178 @@
+             <li class="langn">html</li>
+             <li class="langn">bootstrap</li>
+             <li class="langn" >Ruby</li>
+           </ul>
+           <button class="project-button fullwidth-button">See Project</button>
+       </div>
+       </section>
+
+         <!-- project 5-->
+      <section class="projects" id ="pro-5">
+         <div class = "project-data">
+           <h3 class="project-title">Professional Art Printing Data</h4>
+           <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+           
+           <ul class="languages">
+             <li class="langn">css</li>
+             <li class="langn">html</li>
+             <li class="langn">bootstrap</li>
+             <li class="langn" >Ruby</li>
+           </ul>
+           <button class="project-button fullwidth-button">See Project</button>
+       </div>
+       </section>
+
+         <!-- project 6-->
+      <section class="projects" id ="pro-6">
+         <div class = "project-data">
+           <h3 class="project-title">Professional Art Printing Data</h4>
+           <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+           
+           <ul class="languages">
+             <li class="langn">css</li>
+             <li class="langn">html</li>
+             <li class="langn">bootstrap</li>
+             <li class="langn" >Ruby</li>
+           </ul>
+           <button class="project-button fullwidth-button">See Project</button>
+       </div>
+       </section>
+    </section>
+
+    <section class="about-me">
+      <div class="about-me-main">
+      <h2 class="section-header">About me</h2>
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <button class="project-button">Get My Resume</button>
+    </div>
+    <hr>
+
+  
+    <div class="lang-wrapper">
+    <img src="images/Rectangle 55.svg"  class="Rectangle">
+    <h3 class="language">Language</h3>
+  </div>
+    <ul class="language-list">
+        <li>Javascript</li> 
+        <li>Ruby</li>
+        <li>Html</li>
+        <li>Css</li>
+    </ul>
+
+    <div class="lang-wrapper framework">
+    <img src="images/Rectangle 56.svg"class="Framework">
+<h3 class="fhead">Frameworks</h3>
+</div>
+
+        <ul class="language-list">
+          <li>Bootstrap</li>
+          <li>Ruby on rails</li>
+          <li>Rspec</li>
+          <li>CopyBara</li>
+          <li>Selinium</li>
+        </ul>
+      </li>
```

Comment:
Please make sure your _**about me section**_ uses `flexbox` for the layout , as mentioned in the requirements _**"Make sure the content of the "About myself" section is aligned with flexbox."**_, This will improve the responsiveness of our site and will also help you become a flex ninja :ninja: